README.md

```diff
@@ -1,4 +1,4 @@
-This is the Stackdriver metadata agent.
+This is the Stackdriver Metadata Agent.
```

Comment:
Can we please lowercase the A in Agent?

Reply:
...no!

Reply:
Pretty please? In our public documentation, we use a lowercase a, https://cloud.google.com/monitoring/kubernetes-engine/customizing

Reply:
...fine.